force-app/emails/lwc/sendEmailsForm/sendEmailsForm.html

```diff
@@ -0,0 +1,133 @@
+                                </span>
+                            </template>
+                            <template
+                                for:each={recipientPills}
+                                for:item="recipient"
+                                lwc:else>
+                                <span
+                                    class="slds-pill recipient-pill"
+                                    key={recipient.id}>
+                                    <span class="slds-pill__icon_container">
+                                        <lightning-icon
+                                            icon-name={recipient.iconName}
+                                            size="small">
+                                        </lightning-icon>
+                                    </span>
+                                    <span class="slds-pill__label">
+                                        {recipient.displayName}
+                                    </span>
+                                </span>
+                            </template>
+                        </template>
+                    </div>
+                </div>
+            </div>
+            <div
+                class="slds-form-element__help"
+                lwc:if={recipientsHelpMessage}>
+                    {recipientsHelpMessage}
+            </div>
+            <button
+                class="slds-button"
+                onclick={showRecipientsModal}
+                lwc:if={hasRecipients}>
+                    {label.emailViewRecipients}
+            </button>
+        </div>
+
+        <lightning-input
+            type="text"
+            label={label.emailSubjectLabel}
+            data-id={label.emailSubjectLabel}
+            onchange={handleSubjectChange}
+            placeholder={label.emailEnterSubject}
+            required>
+        </lightning-input>
+
+        <lightning-input-rich-text
+            formats={richTextAllowedFormats}
+            onchange={handleBodyChange}
+            onblur={handleBodyBlur}
+            label={label.emailMessageLabel}
+            valid={isBodyVisiblyValid}
+            message-when-bad-input={bodyErrorMessage}
+            label-visible
+            required>
+        </lightning-input-rich-text>
+
+        <!-- @todo
+        <lightning-button-group class="slds-m-top_xx-small">
+            <lightning-button-icon
+                icon-name="utility:insert_template"
+                alternative-text="Choose template"
+                tooltip="Choose template"
+                size="small">
+            </lightning-button-icon>
+        </lightning-button-group>
+        -->
+
+        <div
+            class="slds-form-element slds-form-element__readonly"
+            lwc:if={hasRelatedToName}>
+            <span class="slds-form-element__label">
+                {label.emailRelatedToLabel}
+            </span>
+            <div class="slds-form-element__control">
```

Comment:
Would it be hard to add a tooltip here for some contextual guidance? Something like "To include information from the related record in the email body, you can include the field reference in the format {{{!record.MyCustomField__c}}}. If the field reference is invalid or the field value for this record is empty, the email will still send but will be missing this text." (but with the correct syntax and verified behavior ...)

Reply:
I have decided to omit this for now.  As I thought about how to communicate to the user the right way to use the merge syntax, it felt like a tooltip is not adequate for accurately instructing the user how to do this.  The right way to do this is via user documentation, and when we add the user documentation we should come back to this and add a tooltip with a reference to that user documentation.

Maybe more ideally would be to implement a merge field picker UI, which is in the backlog as #69.